server/src/main/java/com/tencent/rss/server/LocalStorageChecker.java

```diff
@@ -132,5 +143,64 @@ boolean checkIsHealthy() {
       }
       return isHealthy;
     }
+
+    boolean checkStorageReadAndWrite() {
+      if (storage.isCorrupted()) {
+        return false;
+      }
+      File checkDir = new File(storageDir, "check");
+      try {
+        checkDir.mkdirs();
+        File writeFile = new File(checkDir, "test");
+        if (!writeFile.createNewFile()) {
+          return false;
+        }
+        try (FileOutputStream fos = new FileOutputStream(writeFile)) {
```

Comment:
And it is better to use nio's FileChannel on both read/write, which provides force() API to bypass pagecache.

Reply:
use fsync？

Reply:
The performance with fsync should be fully tested

Reply:
OK